destreamer.ts

```diff
@@ -82,7 +101,11 @@ async function rentVideoForLater(videoUrls: string[], username: string, outputDi
     console.log('Sorry, i mean "you".');
 
     for (let videoUrl of videoUrls) {
-        await page.goto(videoUrl, { waitUntil: 'networkidle2' });
+        let videoID = videoUrl.split('/').pop() ?? (console.error("Couldn't split the videoID, wrong url"), process.exit(25))
+
+        // changed waitUntil value to load (page completly loaded)
+        await page.goto(videoUrl, { waitUntil: 'load' });

```

Comment:
because networkidle2 waits that there are no more than 2 connections in the span of 0.5 sec and on a slow/unreliable connection it esceeds the 30sec timeout and fails the promise.

Since the AcessToken is embedded in the page since the first response after the login there is no reason to wait longer than for the page to load

Reply:
Ah i see, i agree with the approach. Let's have fire on `load` then.